rotkehlchen/chain/ethereum/interfaces/balances.py

```diff
@@ -230,12 +240,15 @@ def query_balances(self) -> BalancesSheetType:
         for address, events in self.addresses_with_gauge_deposits().items():
             # Create a mapping of a gauge to its token
             for event in events:
-                gauge_address = self.get_gauge_address(event)
-                if gauge_address is None:
+                if (
+                    (gauge_address := self.get_gauge_address(event)) is None or
+                    gauge_address in gauges_to_token
+                ):
                     continue
+
                 gauges_to_token[gauge_address] = event.asset.resolve_to_evm_token()
 
-            balances[address] = self._query_gauges_balances(
+            balances[address] += self._query_gauges_balances(
```

Comment:
Isn't this yet another bugfix? But this is a very very basic part of the balances interface. How come this was not seen before? What am I missing.